data.sql

```diff
@@ -20,3 +20,173 @@ postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full
 postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Bob' AND (animals.name = 'Devimon' OR animals.name = 'Plantmon');
 postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Melody Pond' AND (animals.name = 'Charmander' OR animals.name = 'Squirtle' OR animals.name = 'Blossom');
 postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Dean Winchester' AND (animals.name = 'Angemon' OR animals.name = 'Boarmon');
+
+
+-- Fourth Task
+
+-- VETS
+INSERT INTO vets (
+    name, age, date_of_graduation
+) VALUES (
+    'William Tatcher', 45, '2000-04-23'
+);
+
+INSERT INTO vets (
+    name, age, date_of_graduation
+) VALUES (
+    'Maisy Smith', 26, '2019-01-17'
+);
+
+INSERT INTO vets (
+    name, age, date_of_graduation
+) VALUES (
+    'Stephanie Mendez', 64, '1981-05-04'
+);
+
+INSERT INTO vets (
+    name, age, date_of_graduation
+) VALUES (
+    'Jack Harkness', 38, '2008-06-08'
+);
+
+-- specializations
+INSERT INTO specializations (
+    vet_id, specie_id
+) SELECT v.id, s.id
+    FROM species s
+    INNER JOIN vets v ON v.name = 'William Tatcher' AND s.name = 'Pokemon';
+    
+    
+INSERT INTO specializations (
+    vet_id, specie_id
+) SELECT v.id, s.id
+    FROM species s
+    INNER JOIN vets v ON v.name = 'Stephanie Mendez' AND s.name IN ('Digimon', 'Pokemon');
+    
+    INSERT INTO specializations (
+    vet_id, specie_id
+) SELECT v.id, s.id
+    FROM species s
+    INNER JOIN vets v ON v.name = 'Jack Harkness' AND s.name = 'Digimon';
```

Comment:
- [ ] There is one entry missing here, you have only put 3 entries instead of four. Kindly fix this
-The requirements
![image](https://user-images.githubusercontent.com/81932478/184397131-81262546-1024-4521-83ba-1911349c39f2.png)

Reply:
@Munsa1 I added the two from line 60 -64 since it has similar name.